### PR TITLE
Booking screens header

### PR DESCRIPTION
**Pull request by ljau**
# Description
Send parameters trough booking screens header
Display on next booking screens the data obtained
 
 
  ## Screenshots 
image

### CODE REVIEW

File: src/containers/BookingPassengers/index.js [hunk 1]
```diff
@@ -9,15 +9,25 @@ import {
 } from '../../components';
 import {colors} from '../../library/constants';
 
-export const SelectPassengers = ({navigation}) => {
-  const nextScreenCast = type =>
+export const SelectPassengers = ({route: {params}, navigation}) => {
+  console.log('====================================');
```
Comment (Shadic78) [inline src/containers/BookingPassengers/index.js:13]: Delete the logs if you don't need them more

File: README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+- Documentar en [este archivo](setup/README.md) los pasos necesarios para ejecutar la App
+- [Los commits deben se significativos](https://medium.com/better-programming/you-need-meaningful-commit-messages-d869e44e98d4)
+
+
+## Setup
+[Aquí se describen los pasos necesarios para ejecutar y probar este proyecto](setup/README.md)
+
+## Recursos
+[Para apoyo de tus actividades utiliza esta lista de recursos recomendados](https://github.com/bright-coders/commons/tree/master/topics/resources-react-native)
+
```
Comment (YilKanda) [inline README.md:86]: Why is the readme file in this PR?

File: src/containers/BookingPassengers/index.js (hunk 1 above)
Comment (ljau) [reply src/containers/BookingPassengers/index.js]: Logs deleted

File: src/containers/BookingPassengers/index.js [hunk 3]
```diff
@@ -9,15 +9,22 @@ import {
 } from '../../components';
 import {colors} from '../../library/constants';
 
-export const SelectPassengers = ({navigation}) => {
-  const nextScreenCast = type =>
+export const SelectPassengers = ({route: {params}, navigation}) => {
+  
+  const nextScreenCast = () =>
     navigation.navigate('Received', {
```
Comment (YilKanda) [inline src/containers/BookingPassengers/index.js:15]: It is not recommended to upload the console.log() to GitHub, you can use some other option such as Alert. What do you think?

File: README.md (hunk 2 above)
Comment (ljau) [reply README.md:86]: I deleted it by mistake some time ago, I just wanted to restore it so I added it to a PR

**Pull request merged**